src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form" data-qa="big">
+      <input
+        type="search"
+        data-qa="keypress"
+        class="backgorund backgorund__large-bar"
+        name="big-field"
+        placeholder="Try &quot;Los Angeles&quot"
+      >
+      <img
+        src="images/Search.svg"
+        alt="search icon"
+        class="backgorund__large-icon"
+        >
```

Comment:
No need to tag img here, check it on the checklist
<img width="482" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/96fb29dd-06c0-4a15-b17b-4911391ceb19">